### DISCUSSION TITLE
¿Cómo puedo adquirir MATIC?

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/provida#7] [Q&A]
¿Cómo puedo adquirir MATIC?

**Comment by ruth-ga** [comment]
Hasta el momento, la mayoría de las transacciones resultan un poco caras si el monto de las transacciones son bajas. Por lo tanto me parece que el medio más rápido y seguro está haciendo la compra directamente desde nuestra Billetera con tarjeta Visa (a veces acepta tarjeta de débito)
